ColorHighlighter.py

```diff
@@ -1241,6 +1242,37 @@ def get_vars(self, view):
                     if color_vars_file is not None:
                         color_vars_files.append(color_vars_file)
 
+                    # If not opened as a project
+                    if wnd.project_file_name() is None:
+                        # Path of .colorhighlighter file based on folders path
+                        sublime_project = pdata.get("folders", {})[0]["path"] + "/.colorhighlighter"
+                        sublime_project_data = None
+
+                        # If .colorhighlighter file file in path exists
+                        if os.path.isfile(sublime_project):
+                            # Read file
+                            with open(sublime_project) as contents:
+                                # Try to parse JSON
+                                try:
+                                    sublime_project_data = json.load(contents)
+                                except Exception:
+                                    pass
+
+                            # If valid JSON
+                            if sublime_project_data is not None:
+                                # If "color_variables_files" key exists
```

Comment:
This comments and the ones below are quite useless. Comments should be used to explain why something is done, not, what. What is done is pretty clear from the code.